kiarash/JS/JS: this/index.js

```diff
@@ -0,0 +1,13 @@
+function tryThis() {
+  console.log(this);
+}
+
+const thisWrapper = {
+  tryThis,
+};
+
+tryThis.call(thisWrapper);
```

Comment:
This is true, but try to change it and log `thisWrapper` with a different method.